### DISCUSSION TITLE
`[DEP0170] DeprecationWarning: The URL ... is invalid.` error when running `auto version`

### DISCUSSION THREAD

**Discussion by maxsalles** [intuit/auto#2433] [Q&A]
When I try to run `auto version` in the monorepo that I'm working on, I always face the following error:

```
(node:680) [DEP0170] DeprecationWarning: The URL http://git@github.com:{{COMPANY NAME}}/{{PROJECT NAME}}.git is invalid. Future versions of Node.js will throw an error.
(Use `node --trace-deprecation ...` to show where the warning was created)
```

I can't find any explicit reference to URL `http://git@github.com:{{COMPANY NAME}}/{{PROJECT NAME}}.git` in project code base.

Any idea what could be causing this error?

PS.: auto version in the project: `11.0.4`.

**Comment by hipstersmoothie** [comment]
It's probably here would accept a PR!

![CleanShot 2024-02-23 at 12 45 17](https://github.com/intuit/auto/assets/1192452/9dcdeeba-1a05-4c17-877f-3f3ef02ccbd2)

**Comment by ghost** [comment]
I'm seeing the same error above but also followed by a 
`RequestError [HttpError]: Bad credentials` which might be an octokit issue as well? My tokens are fresh and do have the right permissions and valid when authenticating via other means.

**Comment by earonesty** [comment]
its probably this issue, not addressed in node 20.  https://github.com/npm/cli/issues/6869
